src/Halogen/Svg/Core.purs

```diff
@@ -1,21 +1,34 @@
-module Halogen.Svg.Core where
+module Halogen.Svg.Core
+  ( ns
+  , element
+  , attr
+  ) where
 -- Like Halogen.HTML.Core
 
 import Prelude
-import Data.Maybe (Maybe(..))
-import Halogen.HTML.Core (HTML, Prop(Attribute), Namespace(Namespace), AttrName(AttrName))
+import Data.Maybe (Maybe(Just, Nothing))
```

Comment:
This hsould still be `Maybe(..)`

Reply:
Fixed with 269c75f05b9ed91f07baa020041933dd549c21b8